v2/googlecloud-to-elasticsearch/docs/BigQueryToElasticsearch/README.md

```diff
@@ -70,23 +71,30 @@ echo '{
               "isOptional":false
           },
           {
-              "name":"targetNodeAddresses",
-              "label":"Comma separated list of Elasticsearch target nodes",
-              "helpText":"Comma separated list of Elasticsearch target nodes to connect to, ex: http://my-node1,http://my-node2",
+              "name":"connectionUrl",
+              "label":"Elasticsearch URL in format http://hostname:[port] or Base64 encoded CloudId",
+              "helpText":"Elasticsearch URL in format http://hostname:[port] or Base64 encoded CloudId",
```

Comment:
help text is `Elasticsearch URL in the format https://hostname:[port] or specify CloudID if using Elastic Cloud`